examples/hcp-ec2-demo/main.tf

```diff
@@ -78,8 +78,16 @@ module "aws_ec2_consul_client" {
   install_demo_app         = var.install_demo_app
   root_token               = hcp_consul_cluster_root_token.token.secret_id
   security_group_id        = module.aws_hcp_consul.security_group_id
+  ssh_key                  = var.ssh ? tls_private_key.ssh.private_key_pem : ""
   ssh_keyname              = var.ssh ? aws_key_pair.hcp_ec2[0].key_name : ""
   ssm                      = var.ssm
   subnet_id                = module.vpc.public_subnets[0]
   vpc_id                   = module.vpc.vpc_id
 }
+
+module "hashicups" {
```

Comment:
This probably needs a `depends_on` clause for the `aws_ec2_consul_client` module?

Reply:
I think that was implicitly there (because the nomad provided depended on the EC2 public IP), but I'm not 100%. I like that `depends_on` is more explicit, I just made the switch